api/src/http.rs

```diff
@@ -416,6 +418,27 @@ impl Default for ProxyConfig {
     }
 }
 
+/// Configuration information for mirror.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+#[serde(default)]
+pub struct MirrorConfig {
+    /// Dragonfaly http scheme, either 'http' or 'https'.
+    pub scheme: String,
+    /// Dragonfaly url host.
+    pub host: String,
+    /// Request header for Dragonfaly server. Currently, only "X-Dragonfly-Registry" needed.
+    pub header: DragonfalyHeader,
+}
+
+/// Configuration information for Dragonfaly header.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+#[serde(default)]
+pub struct DragonfalyHeader {
```

Comment:
Headers should be transparent to nydusd/backend. They are just extra KV information that nydus has to put together with each GET request. So it's better that we don't define any `Dragonfly` specific struct here.

Reply:
fixed